### DISCUSSION TITLE
SPA mode

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [nuxt/nuxt#16609] [Questions]
Good day,

im building a small mobile app, using capacitor.js
capacitor needs the application to be in SPA form, 
i have set `ssr: false` in nuxt config but, this still creates a server directory inside .output

how can i do this in nuxt3 in nuxt2 this was quite easy to do but i only get a blank page when the app is started in emulator.
with some errors like 

```
E/Capacitor/Console: File: http://localhost/ - Line 581 - Msg: ReferenceError: globalThis is not defined
E/Capacitor: JavaScript Error: {"type":"js.error","error":{"message":"Uncaught ReferenceError: globalThis is not defined","url":"http://localhost/_nuxt/entry.34fa1133.js","line":1,"col":77619,"errorObject":"{}"}}
``` 

also is there a way to disable nitro completely ..?

kind regards.

**Comment by TheAlexLichter** [comment]
Do you build your app via `nuxi generate`?
